README.md

```diff
@@ -1,5 +1,50 @@
-# books-store
+# Bookstore
+- The Bookstore is a website similar to the "Awesome Books" website built in the previous module. It allows a user to:
+  - Display a list of books.
+  - Add a book.
+  - Remove a selected book.
 
- > The Bookstore is a website similar to the "Awesome Books" website built in the previous module. I created an MVP version of it that allows you to:  Display a list of books.
-- [ ] Add a book.
-- [ ] Remove a selected book.
+## Live Demo
+> Live Demo [click me](https://dreamy-villani-473534.netlify.app/?)
+
+### Built With
+
+- [React](https://es.reactjs.org/)
+- [HTML](https://www.w3schools.com/html/)
+- [CSS](https://www.w3schools.com/css/)
+- [JS](https://www.javascript.com/)
+
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+### Installation
+
+1. Clone the repository
+   ```sh
+   git clone https://github.com/mwanawabamgona/book-store.git
+   ```
+2. Install NPM packages
+   ```sh
+   npm install
+   ```
+3. Run server
+   ```sh
+   npm start
+   ```
+## Author
+
+👤 **Ellis Ngona**
+
+- GitHub: [@mwanawabangona](https://github.com/mwanawabangona)
+- Twitter: [@Ellis_aah](https://twitter.com/ellis_ahh)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
```

Comment:
- [OPTIONAL] Please it will be nice to give descriptive names to your feature branches, try to implement that in your future projects.

Reply:
@Mihndim2020 thank you for making me improve my code